### PR TITLE
Convert to kirkstone and weston wayland with etnaviv graphic

### PR DESCRIPTION
**Pull request by RikardO-HM**

### CODE REVIEW

File: conf/conf-notes.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Host Mobility targets are:
     1. console-hostmobility-image *console only*
     2. mobility-image *Graphic based console window and audio*
-    3. mobility-image-xfce *Desktop example version*
+    3. mobility-image-chromium *weston Wayland with chromium webbrowser*
```
Comment (albsod) [inline conf/conf-notes.txt:4]: "Weston desktop with Chromium web browser"

File: conf/conf-notes.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Host Mobility targets are:
     1. console-hostmobility-image *console only*
     2. mobility-image *Graphic based console window and audio*
```
Comment (albsod) [inline conf/conf-notes.txt:3]: How is this different from target 3 below? What graphics is being used?

**Review by albsod — COMMENTED** [review]
Nice work, especially making this work with a modern graphics stack.

File: conf/bblayers.conf.sample [hunk 3]
```diff
@@ -19,5 +19,7 @@ BBLAYERS = " \
   ${BSPDIR}/sources/meta-freescale-distro \
   \
   ${BSPDIR}/sources/meta-hostmobility-bsp \
+  ${BSPDIR}/sources/meta-browser/meta-chromium \
+  ${BSPDIR}/sources/meta-clang \
```
[the remark this answers is not in the document]
Comment (RikardO-HM) [reply conf/bblayers.conf.sample:22]: I dont tink it is needed, but in the future i will try to add bblayer seperate

**Pull request merged**